docs/features/advanced-usage/scores.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Scores"
+description: "Score your requests and experiments"
+---
+
+import QuestionsSection from "/snippets/questions-section.mdx";
+
+<Info>
+  **Who can use this feature**: Anyone on any
+  [plan](https://www.helicone.ai/pricing).
+</Info>
+
+## Introduction
+
+Helicone's scores API allows you to score your requests and experiments. You can use this feature to evaluate the performance of your prompts and compare the scores of different experiments and datasets.
```

Comment:
Helicone's scores API allows you to **evaluate** the performance of your prompts and compare the outcomes of your experiment across different datasets and models. 

does this accurately describe what scores are? I'm trying to rephrase the sentence so that we don't use the word "scores" to define scores lol